src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php

```diff
@@ -66,7 +66,7 @@ public function loadChoiceList($value = null)
             return $this->choiceList;
         }
 
-        return $this->choiceList = new ArrayChoiceList($this->getTimezones(), $value);
+        return $this->choiceList = new ArrayChoiceList(self::getTimezones(), $value);
```

Comment:
Right.I cant imagine this is done on purpose.. at least core doesnt seem to rely on it elsewhere.

Nice catch btw :)